### ISSUE TITLE
Writing to File is broken for me in 4.24

### ISSUE THREAD

**Issue by PoseAI** [ufna/VaRest#257]
Hi, I updated a project to 4.24 and now when I use VA Rest to write a JSON object to file it writes a blank object.  The same blueprint seemed to work fine in 4.23, only change I can see is that VA rest's create json object is no longer pure.  The object is being created and filled properly (I can print them when I encode them as strings) but doesn't seem to write properly anymore.

**Comment by ufna** [comment]
Hi @PoseAI ,

Could you provide a screenshot of BP please?

**Comment by PoseAI** [comment]
sorry I already reverted to 4.23 because I didn't really need to go to 4.24 and it works fine in the previous version.  I figured it was a straightforward bug (I also was surprised that create json object was no longer pure)… I only use the plugin as a quick and easy blueprint json writer.

Only other thing I can add is that a text file is still created by the node but in 4.24 it has no data, only empty fields with comma separation.  I called a debug node print of the string to check just before the write node file and verified that there was data, it just didn't make it to the file.
Hope that helps.   I will probably stick with 4.23 for awhile now.
cheers
